docs/user-guide/SteerLM.rst

```diff
@@ -74,6 +80,16 @@ Untar the .nemo file to obtain the tokenizer in NeMo format (only for the 70B mo
 
 The prefix for the tokenizer would be different when extracted. Ensure that the correct tokenizer file is used when running the preceding command.
 
+**NeMo Megatron Launcher method for Step 1 :**
```

Comment:
I built the doc and didn't like the format that much. the nemo megatron launcher is too embedded into the step by step tutorial. 

can you do the following:

1. make a subsection that is called `Train a SteerLM Model step by step` -> this should have all the existing steps currently in the doc 
2. make a subsection after the above called `Train a SteerLM using the NeMo-Megatron Launcher` this should have your files